upstairs/src/lib.rs

```diff
@@ -6192,6 +6250,30 @@ impl Guest {
         Ok(self.send(wio))
     }
 
+    // Guest does support write_unwritten
```

Comment:
It should :)

Reply:
I put this comment there in part because I get lost as to where I am and this reminds me I am in the Guest Impl.